UI/CombatReport/GraphicalSummary.cpp

```diff
@@ -165,7 +189,17 @@ class BarSizer {
 
     void SetAvailableSize(GG::Pt size) {
         m_available_space = size;
-    }
+
+        m_available_side_bar_space =
+            m_available_space -
+            GG::Pt(BEVEL_MARGIN_X, static_cast<int>(m_summaries.size() + 1) * SIDE_BOX_MARGIN);
+
+        m_available_participant_bar_height = std::max(
+            GG::Y0,
+            m_available_side_bar_space.y -
+            (AXIS_HEIGHT + X_AXIS_LABEL_MARGIN + PARTICIPANT_BAR_UP_MARGIN) *
```

Comment:
preferably indent this and subsequent lines that are part of the same expression, to distinguish it from independent expressions (eg. before the , )